DFC.Digital/DFC.Digital.Services.SendGrid/stylecop.json

```diff
@@ -0,0 +1,16 @@
+﻿{
```

Comment:
This file should not be added as is! This should only be a link to existing file in solution.